Pipeline/PackageBuildOutputs/README.md

```diff
@@ -24,28 +24,31 @@ This section provides a more detailed explanation of how the PackageBuildOutputs
    1. Read [command line parameters](#command-line-options-summary---packagebuildoutputs).
    1. Read the properties file that is passed via `--packagingPropertiesFile`.
 
-1. **Process the DBB build report(s)**
+2. **Process the DBB build report(s)**
    1. If one or multiple DBB build reports are passed to the script via either `--buildReportOrder` or `--buildReportOrderFile`, the script loops through the provided DBB build reports. If no build report is specified, the script reads DBB's `BuildReport.json` file from the pipeline work directory specified by the `--workDir` parameter. For each build report, the following steps are performed:
       1. Parse and extract build output information for records of type *ExecuteRecord* and *CopyToPDSRecord*.
       2. Remove output entries that have no `deployType` set and remove unwanted outputs such as outputs with the `deployType` equal to `ZUNIT-TESTCASE`.
-   2. If processing multiple build reports, a cumulative hashmap of output records is created to be able to combine outputs from multiple pipeline builds into a single tar file.
+   2. If processing multiple build reports, a cumulative hashmap of output records is created to be able to combine outputs from multiple pipeline builds into a single TAR file.
    	  1. The key of the map, used in the calculation of the artifacts to be deployed, is the combination of the member name and the deploy type.
    	  2. Artifacts having the same member name and the same deploy type will be present only once in the generated package, taking the last occurrence of the artifact, as found in the ordered list of Build Reports passed as parameters.
 
-1. **(Optionally) Generate Wazi Deploy application manifest **
+3. **(Optionally) Generate Software-Bill-Of-Material (SBOM) file **
+   1. Based on the collected build outputs information, an SBOM file following the [CycloneDX](https://cyclonedx.org/) specification is created.  
+   More details can be found [in this section](#software-bill-of-material-sbom-generation).
+
+4. **(Optionally) Generate Wazi Deploy application manifest **
```

Comment:
Same as above. If you could fix this (from my previous PR) `fest **`, thank you :-)